api.js

```diff
@@ -14,6 +14,8 @@
  *   limitations under the License.
  */
 
+require('es6-shim');
```

Comment:
This probably needs to be in https://github.com/jspm/jspm-cli/blob/master/cli.js as well? Or every file?